packages/jwt/test/jwt.test.ts

```diff
@@ -1,80 +1,81 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { Wallet } from 'ethers';
+import jsonwebtoken from 'jsonwebtoken';
+import ECKey from 'ec-key';
 import { Keys } from '@ew-did-registry/keys';
-import { JWT } from '../src';
+import { JWT } from '../src/JWT256K';
+import { Algorithms } from '../src/types';
 
 const { expect, should } = chai;
-chai.use(chaiAsPromised);
 
+chai.use(chaiAsPromised);
 should();
 
-const keyPairAlice = new Keys({ privateKey: '635a1ba32ab7218f6df7785c8e128f485c612cdb68cf12a1157987130e7e6d0d' });
-
-const jwtAlice = new JWT(keyPairAlice);
 const payload = { claim: 'test' };
 
-let token: string;
-let jwt: JWT;
-
-const testSuite = (): void => {
-  it('jwt signed with private key should return a string', async () => {
-    expect(token).to.be.a('string');
-  });
-
-  it('verified jwt should return the payload', async () => {
-    const BobKeyPair = new Keys({ privateKey: '63511ba32ab7218f6df7785c8e128f485c612cdb68cf12a1157987130e7e6d0d' });
-    const jwtBob = new JWT(BobKeyPair);
-
-    let decoded;
-
-    try {
-      decoded = await jwtBob.verify(token, keyPairAlice.publicKey);
-    } catch (e) {
-      console.log(e);
-    }
-    expect(decoded).to.eql(payload);
+describe('[JWT PACKAGE]', () => {
+  let signerPubKey: string;
+  let signer: JWT;
+  let token: string;
+  let verifier: JWT;
+  let algorithm: Algorithms;
+
+  const testSuite = () => {
+    it('jwt signed with private key should return a string', async () => {
+      expect(token).to.be.a('string');
+    });
+
+    it('signed token should be verified', async () => {
+      expect(verifier.verify(token, signerPubKey, { algorithms: [algorithm] })).to.eql(payload);
+    });
+
+    it('verification with wrong signature should throw error', async () => {
+      return expect(function () { verifier.verify(token, new Keys().publicKey, { algorithms: [algorithm] }); }).to.throw();
```

Comment:
Maybe we can shorten a little bit the size of line ?
```suggestion
      expect(function () { verifier.verify(token, new Keys().publicKey, { algorithms: [algorithm] }); }).to.throw();
```